README.md

```diff
@@ -1,5 +1,5 @@
 # AccessLocator
 
-The web application source files are in app/.  This is an application using the Laravel PHP framework.  See its readme for details.
+The web application source files are in app/.  This is an application using the Laravel PHP framework.  Check out readme for details.
```

Comment:
It looks like this commit came from the practice pull request you made.  

Since it isn't related to the issue, completely remove the commit responsible for this change.  Don't just revert it.

Let me know if you have questions on how to remove the commit.  

In case you need ideas, here is a command sequence that should do it:
```
git checkout master
git reset HEAD~10 --hard
git pull upstream master
git checkout dev-parth
git rebase -i master
```
A text editor will open listing the commits on your branch.  The top commit will be something like:
```
pick 12345abcd Practice commit from Parth Patel.
```
Remove that single line and then save and close the text editor.

If it successfully rebases, push up again with something like "git push origin dev-parth -f"

If you want more general documentation on using git with AccessLocator, there is a wiki at https://github.com/hhaccessibility/hhaccessibility.github.io/wiki/git-commands.